pandas-stubs/core/reshape/pivot.pyi

```diff
@@ -33,12 +40,12 @@ def pivot(
     values: IndexLabel = ...,
 ) -> DataFrame: ...
 def crosstab(
-    index: Sequence | Series,
-    columns: Sequence | Series,
-    values: Sequence | None = ...,
-    rownames: Sequence | None = ...,
-    colnames: Sequence | None = ...,
-    aggfunc: Callable | None = ...,
+    index: list | npt.NDArray | Series | list[Sequence | npt.NDArray | Series],
+    columns: list | npt.NDArray | Series | list[Sequence | npt.NDArray | Series],
+    values: list | npt.NDArray | Series | None = ...,
+    rownames: list[HashableT] | None = ...,
+    colnames: list[_HashableT2] | None = ...,
+    aggfunc: str | np.ufunc | Callable[[Series], float] | None = ...,
```

Comment:
Looking at the code, `aggfunc` is `AggFuncType`

Reply:
AggFuncType isn't specific enough (also AggFuncType is no longer part of pandas-stubs).  The input to the aggfunc is always a Series and the output is a scalar (int or float in the standard crosstab case).

Reply:
OK, I see what you're saying.